qa/tasks/cephadm.py

```diff
@@ -642,9 +642,9 @@ def ceph_mdss(ctx, config):
     yield
 
 @contextlib.contextmanager
-def ceph_prometheus(ctx, config):
+def ceph_generic(daemon_type, ctx, config):
```

Comment:
is this generic, or monitoring specific?

```suggestion
def ceph_monitoring(daemon_type, ctx, config):
```

Reply:
I think it makes sense to call it `ceph_monitoring`.